### PR TITLE
update k8s version to v1.21.8

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Now KubeEdge supports k8s v1.21, So try to update CI kubelet version and supported k8s version defined in codes.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by fisherxu — COMMENTED** [review]
The code is 1.21.4?

**Comment by gy95** [comment]
> The code is 1.21.4?

kind@v0.11.1 is based on v1.21.1 : https://github.com/kubernetes-sigs/kind/releases
I think we can upgrade it to the latest v1.21 version, so I chose v1.21.8

File: common/constants/default.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ const (
 	DefaultVolumeStatsAggPeriod        = time.Minute
 	DefaultTunnelPort                  = 10004
 
-	CurrentSupportK8sVersion = "v1.19.3"
```
Comment (fisherxu) [inline common/constants/default.go:67]: This means the k8s version in KubeEdge which is v1.21.4 now, we can cherry-pick to release-1.9.

Comment (gy95) [reply common/constants/default.go:67]: got it, it's from vendor version.

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Comment by fisherxu** [comment]
Please cherry-pick it to release-1.9 :)

**Pull request merged**